streams/src/main/java/org/apache/kafka/streams/state/internals/metrics/RocksDBMetrics.java

```diff
@@ -0,0 +1,382 @@
+    private static final String MEMTABLE_FLUSH_TIME = "memtable-flush-time";
+    private static final String MEMTABLE_FLUSH_TIME_AVG = MEMTABLE_FLUSH_TIME + AVG_SUFFIX;
+    private static final String MEMTABLE_FLUSH_TIME_MIN = MEMTABLE_FLUSH_TIME + MIN_SUFFIX;
+    private static final String MEMTABLE_FLUSH_TIME_MAX = MEMTABLE_FLUSH_TIME + MAX_SUFFIX;
+    private static final String WRITE_STALL_DURATION = "write-stall-duration";
+    private static final String BLOCK_CACHE_DATA_HIT_RATIO = "block-cache-data-hit" + RATIO_SUFFIX;
+    private static final String BLOCK_CACHE_INDEX_HIT_RATIO = "block-cache-index-hit" + RATIO_SUFFIX;
+    private static final String BLOCK_CACHE_FILTER_HIT_RATIO = "block-cache-filter-hit" + RATIO_SUFFIX;
+    private static final String BYTES_READ_DURING_COMPACTION = "bytes-read-compaction";
+    private static final String BYTES_WRITTEN_DURING_COMPACTION = "bytes-written-compaction";
+    private static final String COMPACTION_TIME = "compaction-time";
+    private static final String COMPACTION_TIME_AVG = COMPACTION_TIME + AVG_SUFFIX;
+    private static final String COMPACTION_TIME_MIN = COMPACTION_TIME + MIN_SUFFIX;
+    private static final String COMPACTION_TIME_MAX = COMPACTION_TIME + MAX_SUFFIX;
+    private static final String NUMBER_OF_OPEN_FILES = "number-open-files";
+    private static final String NUMBER_OF_FILE_ERRORS = "number-file-errors";
+
+    private static final String BYTES_WRITTEN_TO_DB_RATE_DESCRIPTION =
+        "Average number of bytes written per second to the RocksDB state store";
+    private static final String BYTES_WRITTEN_TO_DB_TOTAL_DESCRIPTION =
+        "Total number of bytes written to the RocksDB state store";
+    private static final String BYTES_READ_FROM_DB_RATE_DESCRIPTION =
+        "Average number of bytes read per second from the RocksDB state store";
+    private static final String BYTES_READ_FROM_DB_TOTAL_DESCRIPTION =
+        "Total number of bytes read from the RocksDB state store";
+    private static final String MEMTABLE_BYTES_FLUSHED_RATE_DESCRIPTION =
+        "Average number of bytes flushed per second from the memtable to disk";
+    private static final String MEMTABLE_BYTES_FLUSHED_AVG_DESCRIPTION =
+        "Total number of bytes flushed from the memtable to disk";
+    private static final String MEMTABLE_HIT_RATIO_DESCRIPTION =
+        "Ratio of memtable hits relative to all lookups to the memtable";
+    private static final String MEMTABLE_FLUSH_TIME_AVG_DESCRIPTION =
+        "Average time spent on flushing the memtable to disk in ms";
+    private static final String MEMTABLE_FLUSH_TIME_MIN_DESCRIPTION =
+        "Minimum time spent on flushing the memtable to disk in ms";
+    private static final String MEMTABLE_FLUSH_TIME_MAX_DESCRIPTION =
+        "Maximum time spent on flushing the memtable to disk in ms";
+    private static final String WRITE_STALL_DURATION_AVG_DESCRIPTION = "Moving average duration of write stalls in ms";
```

Comment:
`Moving average duration` may be a bit confusing to readers, maybe just `Average duration of ..`.